Doppler.Currency/Startup.cs

```diff
@@ -115,7 +116,13 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerF
         {
             app.UseStaticFiles();
 
-            loggerFactory.AddFile("Logs/app-{Date}.log");
+            var configuration = new ConfigurationBuilder()
+                .AddJsonFile("appsettings.json")
+                .Build();
+
+            Log.Logger = new LoggerConfiguration()
+                .ReadFrom.Configuration(configuration)
```

Comment:
Is it not possible reading the current configuration in place of reading `appsettings.json` again?

I am saying this because there are other configuration sources, for example, environment variables, environment specific files, secrets, etc. 

@elsupergomez how are you doing this in other projects?

Reply:
@andresmoschini do you you say this? [doppler-mailer-system](https://github.com/MakingSense/doppler-mailer-system/blob/develop/Doppler.MailerSystem.IncomingMessageAnalyzer/Program.cs#L21) from Program.cs, I can remove the modification in Startup.cs with this approach
`.UseSerilog((hostContext, loggerConfiguration) => loggerConfiguration.ReadFrom.Configuration(hostContext.Configuration))`
@elsupergomez

Reply:
I am not familiar with that. Rodrí would help you better.

Reply:
Yes is better configure in `CreateHostBuilder` method with this approach.

```.UseSerilog((hostContext, loggerConfiguration) => loggerConfiguration.ReadFrom.Configuration(hostContext.Configuration))``` 

this will use any configuration source.